src/Support/Contracts/Directive.php

```diff
@@ -9,5 +9,5 @@
      *
      * @return string
      */
-    public function name();
+    public function name(): string;
```

Comment:
This is a public facing interface, let's remove the required return type here.

Reply:
Did it.